### DISCUSSION TITLE
Mutable layer number for autoencoder

### DISCUSSION THREAD

**Discussion by zyfccc** [microsoft/nni#4432] [Q&A]
In retiarii, I wonder whether there is a way to mutate the layer number of an autoencoder with a fixed number of latent variables? For example, an autoencoder with one, two, or three hidden layers of varied number of neurons in each layer and a latent variable. 

A code example would be nice.

Thanks

**Comment by Richard-gangan** [comment]
I have the same question. How can can I find the best layer number and the appropriate number of neurons. But I cant run the example of NNI now. Maybe we can communicate about it?
